enable spelling and boilerplate check

**Is your feature request related to a problem?/Why is this needed**
<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->

**Describe the solution you'd like in detail**
<!-- A clear and concise description of what you want to happen. -->
https://github.com/kubernetes-csi/csi-driver-nfs/pull/127 disabled following two checks:
 - verify-spelling.sh
 - verify-boilerplate.sh

need to upstream the above two scripts to https://github.com/kubernetes-csi/csi-release-tools.git and then re-enable these two scripts in this project

**Describe alternatives you've considered**
<!-- A clear and concise description of any alternative solutions or features you've considered. -->

**Additional context**
<!-- Add any other context or screenshots about the feature request here. -->

```
error: docs/csi-dev.md:38:12: "environmnet" is a misspelling of "environments"
error: docs/csi-dev.md:89:6: "environmnet" is a misspelling of "environments"
error: release-tools/SIDECAR_RELEASE_PROCESS.md:14:17: "maintainence" is a misspelling of "maintenance"
error: release-tools/prow.sh:76:31: "seperately" is a misspelling of "separately"
error: release-tools/prow.sh:296:37: "seperated" is a misspelling of "separated"
error: release-tools/prow.sh:1001:32: "succesful" is a misspelling of "successful"
Found spelling errors!
Cleaning up...
Makefile:46: recipe for target 'verify' failed
make: *** [verify] Error 1
Error: Process completed with exit code 2.
```

--- a/.prow.sh
+++ b/.prow.sh
@@ -22,4 +22,7 @@
 
 . release-tools/prow.sh
 
+./release-tools/verify-boilerplate.sh "$(pwd)"
+./release-tools/verify-spelling.sh "$(pwd)"
+
 main

--- a/docs/csi-dev.md
+++ b/docs/csi-dev.md
@@ -35,7 +35,7 @@ $ cd $GOPATH/src/github.com/kubernetes-csi/csi-driver-nfs
 $ ./_output/nfsplugin --endpoint tcp://127.0.0.1:10000 --nodeid CSINode -v=5 &
 ```
 
-#### 0. Set environmnet variables
+#### 0. Set environment variables
 ```console
 $ cap="1,mount,"
 $ volname="test-$(date +%s)"
@@ -86,7 +86,7 @@ CSINode
 ```
 
 ## How to test CSI driver in a Kubernetes cluster
-- Set environmnet variable
+- Set environment variable
 ```console
 export REGISTRY=<dockerhub-alias>
 export IMAGE_VERSION=latest

--- a/hack/verify-all.sh
+++ b/hack/verify-all.sh
@@ -22,7 +22,7 @@ ${PKG_ROOT}/hack/verify-gofmt.sh
 ${PKG_ROOT}/hack/verify-govet.sh
 ${PKG_ROOT}/hack/verify-golint.sh
 ${PKG_ROOT}/hack/verify-yamllint.sh
-#${PKG_ROOT}/hack/verify-boilerplate.sh
-#${PKG_ROOT}/hack/verify-spelling.sh
+${PKG_ROOT}/hack/verify-boilerplate.sh
+${PKG_ROOT}/hack/verify-spelling.sh
 ${PKG_ROOT}/hack/verify-helm-chart.sh
 ${PKG_ROOT}/hack/verify-gomod.sh

